fields/urls.json

```diff
@@ -0,0 +1,33 @@
+{
+  "title": "Website Addresses/URLs",
+  "description": "URLs for website(s), social media, etc.",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "name": {
+        "title": "URL Name",
+        "description": "The name of what this URL is for (e.g., the name of the social media provider, website, etc.)",
+        "type": "string",
+        "minLength": 1,
+        "maxLength": 100
+      },
+      "url": {
+        "$ref": "url.json"
```

Comment:
This should be `primary_url.json`

Reply:
Thanks for catching that @geoffturk. Instead of reffing the `primary_url` field (which could be problematic because the field description might conflict with usage in the multiple URLs field), I've just moved the subfield definition into the `urls` field file itself.